sdks/java/core/src/main/java/org/apache/beam/sdk/coders/Coder.java

```diff
@@ -57,9 +53,13 @@
  * inferred. See {@link KvCoder} for an example of a simple compound {@link Coder} that supports
  * automatic composition in the {@link CoderRegistry}.
  *
+ * <p>A {@link CoderProvider} can be automatically registered with the {@link CoderRegistry} by the
```

Comment:
Suggestion: omit "automatically" since it is not quite true (the user wrote a registrar) and otherwise redundant.